README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## NAME               
+
+My name is Liang __CAI__.          
+
+
+## RESEARCH INTEREST                               
+My research interests include:                   
+1. Urban Sociology                        
+2. Criminology                   
+3. Geography             
+
+## LINK                       
+For more information, please visit my UChicago page, [LINK HERE](https://sociology.uchicago.edu/directory/liang-cai).
```

Comment:
Although you successfully added a link, I think there might be a more elegant way of doing it instead of having the alt text reading "LINK HERE". For example, you could do the following 
" For more information, please visit [my UChicago page](https://sociology.uchicago.edu/directory/liang-cai)." 
If this is your markdown code, what will be hyperlinked is "my UChicago page", which perhaps might look better. Still, good job embedding a link!

Reply:
Whoops, I just realized my writing formatted itself when I posted the comment. 
For the formatting of the embedding of the link, this ought to be the format. 
[my Uchicago Page](your url).

Reply:
I agree that with the above comments about perhaps creating the link around "my UChicago page" rather than "LINK HERE"